Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -24,22 +25,28 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
                 return .Settings.Search.QuickSearchEnginesTitle
             case .privateSession:
                 return .Settings.Search.PrivateSessionTitle
+            case .fxSuggestSettings:
+                return String.localizedStringWithFormat(.Settings.Search.Suggest.AddressBarSettingsTitle, AppName.shortName.rawValue)
             }
         }
     }
 
     private let profile: Profile
     private let model: SearchEngines
 
-    fileprivate let ItemDefaultEngine = 0
-    fileprivate let ItemDefaultSuggestions = 1
-    fileprivate let IconSize = CGSize(width: OpenSearchEngine.UX.preferredIconSize,
-                                      height: OpenSearchEngine.UX.preferredIconSize)
+    private let ItemDefaultEngine = 0
+    private let ItemDefaultSuggestions = 1
+    private let ItemSuggestionFirefox = 0
+    private let ItemSuggestionSponsors = 1
+    private let ItemSuggestionLearn = 2
+    private let IconSize = CGSize(width: OpenSearchEngine.UX.preferredIconSize,
+                                  height: OpenSearchEngine.UX.preferredIconSize)
 
-    fileprivate var showDeletion = false
+    private var showDeletion = false
+    private var sectionsToDisplay: [SearchSettingsTableViewController.Section] = []
```

Comment:
I realized displaying the correct section doesn't work when the felt-privacy feature is disabled and fx-suggest is enabled, because the current logic will show the privacy toggle section. I added this to keep track of which section should be displayed, which is decided in `numberOfSections()`

Reply:
Nice, this makes a lot of sense to me! 🚀